role-name/molecule/roles/verify.yml

```diff
@@ -0,0 +1,9 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: all
+  tasks:
+    - name: Test Description
+      include_role:
+        name: test-role-name
```

Comment:
As above, we could add a placeholder here such as `test-<role-name>` to ensure consistency.